src/lib/BetaBanner.svelte

```diff
@@ -0,0 +1,11 @@
+<div class="govuk-phase-banner">
+  <p class="govuk-phase-banner__content" style="margin-bottom: 0px">
+    <strong class="govuk-tag govuk-phase-banner__content__tag">Beta</strong>
+    <span class="govuk-phase-banner__text">n
+      This is a new tool. Contact <a href="mailto:plan-your-active-travel-schemes@activetravelengland.gov.uk">
```

Comment:
Should we update the email in the alpha component too? And do we want to cutover to this component in this PR yet?